### PR TITLE
Remove Blame parent feature as GitHub has added it

### LINKED ISSUE #208: Blame parent links should link to specific line
If possible the blame parent links should try to link to specific lines which correspond to the code chunk which the blame link applies to. So it shouldn't respect existing line-specific links in the URL `#LX-Y`, but rather link to the lines corresponding to the hunk where the link was clicked.

I'm not sure if I'm explaining it properly. I could provide examples and an animation if needed.

### LINKED ISSUE #345: Blame parent links no longer blame the parent
#248 removed the ^ from the blame parent URL, so it now blames the commit of the line you clicked on, not its parent.

### PR DESCRIPTION
**Pull request by hkdobrev**
See [blog post](https://github.com/blog/2304-navigate-file-history-faster-with-improved-blame-view) and [help documentation](https://help.github.com/articles/tracing-changes-in-a-file/) on the new changes.

Closes #208 
Closes #345

### CODE REVIEW

**Comment by paulmolluzzo** [comment]
PR looks good to me. 👍 

Did we have any part in getting GH to implement this feature. :thinking:

**Comment by hkdobrev** [comment]
> Did we have any part in getting GH to implement this feature. 🤔

I'd like to believe so, but not to my knowledge.

**Comment by hkdobrev** [comment]
@jasonrudolph Hi! I'm asking you here as you have posted the blog post about the new blame functionality. First of all, thank you for your work. It's very much appreciated!

What was the reasoning behind the new feature (btw it's perfectly implemented and it's really fast)? Was it inspired by this humble extension? I know a couple of GitHubbers are using it.

**Comment by paulmolluzzo** [comment]
> I'd like to believe so, but not to my knowledge.

I'm not sure I was seriously asking... 🙈

File: readme.md [hunk 1]
```diff
@@ -18,7 +18,6 @@ Our hope is that GitHub will notice and implement some of these much needed impr
 - [Adds user avatars to Reactions](screenshot-reactions.png)
 - [Adds a quick edit button to the readme](https://cloud.githubusercontent.com/assets/170270/13379292/61cd4c42-de54-11e5-8829-f4b82ba8c2bc.png)
 - [Shows current filename in the sticky pull request header](https://cloud.githubusercontent.com/assets/170270/14153322/97a8e902-f6e1-11e5-8331-19e284e3e6fa.png)
-- [Adds blame links for parent commits in blame view](https://github.com/sindresorhus/refined-github/issues/2#issuecomment-189141373)
```
Comment (sindresorhus) [inline readme.md]: When we remove features GitHub implemented, I think we should instead strike `~~` and put it at the bottom with a comment that it was implemented.

**Comment by jasonrudolph** [comment]
> What was the reasoning behind the new feature (btw it's perfectly implemented and it's really fast)?

@hkdobrev: I'm glad you're enjoying it! :heart_decoration:

I first discovered [reblame in fugitive.vim](https://github.com/tpope/vim-fugitive/blob/b754bc2031f21a532c083dd0d072ba373bbe3a37/doc/fugitive.txt#L220). When I moved from Vim to Atom, I really missed reblame. While I thought that it would be cool and useful to add reblame to Atom at some point (and I still think that :smile:), I also thought it would be really awesome to make reblame available to everyone who uses the GitHub UI. So during a recent internal hack week at GitHub, @jdpace and I put together a prototype and invited GitHub employees to try it out. After getting help from @cobyism to improve the UX, and working out a few kinks, we were ready to share it with the world. :shipit:

By the way, it's super :cool: to see the types of GitHub customizations that people are finding useful via this Chrome extension. I'm already wanting to try out several of them. Things like #206 seem incredibly handy to me. :zap:

**Comment by hkdobrev** [comment]
@jasonrudolph Thanks for stopping by and thank you for your work! I hope this extension is taken in good spirit by GitHub team and inspire more features for everyone. The extension now have 3,600+ users and we hope to bring it to Firefox as well.

File: readme.md (hunk 1 above)
Comment (hkdobrev) [inline readme.md]: @sindresorhus Done.

**Comment by sindresorhus** [comment]
@jasonrudolph Thanks for making this feature. I use ours all the time, and it's nice to finally have it natively in the GitHub UI :)

**Pull request merged**